app.js

```diff
@@ -89,16 +89,20 @@
   }
 
   function updateAQI(sensor) {
-    let pm25s = [];
-    let humidity = sensor.results[0].humidity;
-
+    sensor = sensor.sensor;
     announceState("Calculating AQI");
 
-    for (const subsensor of sensor.results) {
-      if (!bustedSensor(subsensor)) {
```

Comment:
We get a lot less information from PurpleAir now, making this check a lot simpler, so I brought it inline. If we ever want, we can request more data from them and bring this back.